src/Commands/AdminCommand.php

```diff
@@ -2,12 +2,13 @@
 
 namespace TCG\Voyager\Commands;
 
+use App\User;
```

Comment:
No need to include this line.  You're not using it anywhere, and it's not guaranteed to exist.